lib/clearance/password_strategies/blowfish.rb

```diff
@@ -15,7 +15,7 @@ module Blowfish
       #   gem
       def authenticated?(password)
         warn "#{Kernel.caller.first}: #{DEPRECATION_MESSAGE}"
-        encrypted_password == encrypt(password)
+        encrypted_password == _encrypt(password)
```

Comment:
The blowfish password strategy is deprecated. Please do not make any changes to this file. Thanks!